src/js/components/More/PricingCard.jsx

```diff
@@ -0,0 +1,698 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Button from '@material-ui/core/Button';
+import { withStyles } from '@material-ui/core/styles';
+import styled from 'styled-components';
+import Close from '@material-ui/icons/Close';
+import Check from '@material-ui/icons/Check';
+
+class PricingCard extends Component {
+  static propTypes = {
+    premium: PropTypes.bool,
+    bullets: PropTypes.array,
+    price: PropTypes.number,
+    planName: PropTypes.string,
+    priceDescribe: PropTypes.string,
+    description: PropTypes.string,
+    classes: PropTypes.object,
+    buttonText: PropTypes.string,
+    fullWidth: PropTypes.bool,
+  };
+
+  render () {
+    const { premium, bullets, price, planName, priceDescribe, description, classes, buttonText, fullWidth } = this.props;
+
+    return (
+      <React.Fragment>
+        {fullWidth ? (
+          <div className="col col-12">
+            <Card>
+              <CardWrapper>
+                {premium ? (
+                  <PremiumName>{planName}</PremiumName>
+                ) : (
+                  <DefaultName>{planName}</DefaultName>
+                )}
+                {price === 0 || price ? (
+                  <React.Fragment>
+                    <span style={{
+                      fontSize: '18px',
```

Comment:
@DaleMcGrew is it a good idea to allow inline styling?

Reply:
Hello @jarodpeachey, I think @lazorfuzz has a good point. Could you please implement these as a named styled component (like you did with "PriceDescribe" for example) instead of using a span with inline styles?

Reply:
Sure thing, I'm going out  right now, but should have that fixed by tonight!